.appveyor.yml

```diff
@@ -0,0 +1,46 @@
+# To activate, change the Appveyor settings to use `.appveyor.yml`.
+install:
+  - python -m pip install --upgrade virtualenv pip setuptools tox
+
+  # Fetch the three main PyPy releases
+  - ps: (New-Object Net.WebClient).DownloadFile('https://bitbucket.org/pypy/pypy/downloads/pypy-2.6.1-win32.zip', "$env:appveyor_build_folder\pypy-2.6.1-win32.zip")
+  - ps: 7z x pypy-2.6.1-win32.zip | Out-Null
```

Comment:
Despite its very low number, 2.6.1 is not old.  It was released only a year ago.  That series is/was the stable release.  PyPy 2.6.1 is `sys.version_info` 2.7.10, which is the same as the latest PyPy2 5.x release, and it gets those users past the Python SSL problems.  PyPy 5.x appears to be gaining traction as a replacement for PyPy 2.x.

Also, it costs us almost nothing?  The time to add it to the image is negligible.  There are no Windows specific bugs related to PyPy 2.x series; the only Windows specific bug anywhere in pyflakes is https://bitbucket.org/pypy/pypy/issues/2350 , which also occurs on the latest PyPy2 5.x release, and it is only a problem with our test framework -- not the program end-users are faced with.

As we test against Travis `python: pypy`, which is PyPy2.5 (`sys.version_info` 2.7.8), IMO it makes sense to bring our Windows testing to roughly the same level.

Without evidence otherwise, Windows users have been able to use pyflakes on PyPy2.x , and by adding this to the test matrix we ensure that if/when we do break that, we do it intentionally.

I expect we'd break it when Travis stops providing a 2.5/2.6 by default, or the existing pypy 2.5/2.6 workarounds start to really hurt. (afaik, the only painful aspect is the different column offsets in their error messages, which we should raise a bug in pypy so that problem can whittle away over time.)
I expect PyPy3 2.4 will be the first to be killed.  While it has been added to this test matrix, I would be only too happy to remove it entirely just as soon as they provide a -win release of PyPy3 5.x, as it is Python 3.2, and anybody using that in production a second longer than they need to has bigger problems than their syntax checker no longer working.